lively.morphic/morph.js

```diff
@@ -2601,25 +2612,19 @@ export class Morph {
   // -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
   // comments
   // -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
-  async addComment (commentText, relativePosition = pt(0, 0)) {
-    const { CommentData } = await System.import('lively.collab');
-    const comment = new CommentData(commentText, relativePosition);
-    this.comments.push(comment);
-    const commentBrowser = $world.getSubmorphNamed('Comment Browser');
-    if (commentBrowser) commentBrowser.viewModel.addCommentForMorph(comment, this);
-    return comment;
-  }
 
-  removeComment (commentToRemove) {
-    const commentBrowser = $world.getSubmorphNamed('Comment Browser');
-    if (commentBrowser) commentBrowser.viewModel.removeCommentForMorph(commentToRemove, this);
-    this.comments = this.comments.filter(comment => !commentToRemove.equals(comment));
-  }
-
-  emptyComments () {
-    const commentBrowser = $world.getSubmorphNamed('Comment Browser');
-    if (commentBrowser) this.comments.forEach((comment) => commentBrowser.viewModel.removeCommentForMorph(comment, this));
-    this.comments = [];
+  /**
+   * Retrieves comments made on this morph from the world and returns them. Returns an empty array when no comments exist.
+   * @see `lively.collab` for more information on comments.
+   * Comments are only available when the world is a `LivelyWorld` provided by `lively.ide`.
+   * When comments are not available, this method returns an empty array.
+   * @returns {[CommentData]} An array of the comments on this morph or an empty array 
+   */
+  get comments () {
+    if (!$world.isIDEWorld) return [];
```

Comment:
dito

Reply:
I don't think so? We are in `Morph` here...